internal/compose/compose.go

```diff
@@ -318,8 +318,7 @@ func (p *Project) Logs(opts CommandOptions) ([]byte, error) {
 func (p *Project) WaitForHealthy(opts CommandOptions) error {
 	// Read container IDs
 	args := p.baseArgs()
-	args = append(args, "ps")
-	args = append(args, "-q")
+	args = append(args, "ps", "-a", "-q")
```

Comment:
I encountered false positive failures for service containers that successfully finished but did so before reaching this point, so they would not be listed without `-a`. Since `exited but with 0` is considered healthy down the line, this is a requirement for these containers to show.